### PR TITLE
Add script for processing data

### PR DESCRIPTION
**Pull request by richfitz**
Add little script to process ADM1 data to json.

This uses the current version from Claire's files, and I've not committed the resulting files (though can do once we're happy with the format and path).

### CODE REVIEW

File: scripts/process-data.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env Rscript
+process_country <- function(x) {
+  id <- sprintf("%s-%s-%s", x$IS0, x$ID_0, trimws(x$ID_1))
```
Comment (EmmaLRussell) [inline scripts/process-data.R]:
```suggestion
  id <- sprintf("%s.%s_1", x$IS0, trimws(x$ID_1))
```

**Review by EmmaLRussell — COMMENTED** [review]
This works great thanks. I think the ID generation needs tweaking a little. Talked this over with Wes yesterday and we're going to try to use the GID_1 and GID_2 IDs from the GADM data,. As far as I can tell, GID1 made up of the  ISO, the region id and the level (but not the numeric id of the country, ie ID_0). But we can always fiddle with that later if required.

**Comment by richfitz** [comment]
Do you want me to commit the generated files too?

**Comment by EmmaLRussell** [comment]
> Do you want me to commit the generated files too?

No that's alright, I think I'll just add them in a branch together with  Wes's geo files and code tweaks (hopefully just tweaks!) to get them all working together.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**